src/main/java/org/zalando/nakadi/partitioning/StringHash.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.nakadi.partitioning;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class StringHash {
+
+    public int hashCode(final String string) {
```

Comment:
Just to mention here, that this function will return the same hash codes for "Aa" and "BB".
Probably, we are not allowed to change it.